android/src/main/java/com/RNFetchBlob/RNFetchBlob.java

```diff
@@ -120,11 +120,8 @@ public void actionViewIntent(String path, String mime, final Promise promise) {
                 // Set flag to give temporary permission to external app to use FileProvider
                 intent.setFlags(Intent.FLAG_GRANT_READ_URI_PERMISSION);
 
-                // Validate that the device can open the file
-                PackageManager pm = getCurrentActivity().getPackageManager();
-                if (intent.resolveActivity(pm) != null) {
-                    this.getReactApplicationContext().startActivity(intent);
-                }
+                // Do not validate that the device can open the file - instead let it throw
```

Comment:
I think the final comment will lack context. I suggest:

`// Open the file, throws if it cannot`